### ISSUE TITLE
Model example validation fails on lists of items that have refs

### ISSUE THREAD

**Issue by theldberg** [OpenAPITools/openapi-style-validator#169]
**Describe the bug**
Model example validation fails on lists of items that have refs. The same case works if it is a single object property with a ref.

For example, this passes:

```
mapData:
          $ref: "#/components/schemas/MapData"
```

This fails:

```
airlines:
  type: array
  items:
    $ref: "#/components/schemas/Airline"

```
Both of those properly render valid examples in swagger.

**To Reproduce**
1.) Create a swagger document with a schema reference
2.) There should be at least one property that is an array of objects that uses a ref
3.) Run validation with the default settings
4.) It tells you that you are missing examples

I've attached a yaml that shows the issue. Validation on the single property passes but the array property fails, when both properly render examples in swagger.

[test_examples.yaml.txt](https://github.com/OpenAPITools/openapi-style-validator/files/6024290/test_examples.yaml.txt)

**Expected behavior**
No validation errors occur if an item list renders an appropriate example in swagger.

**Additional context**
I attempted to do a PR but am getting a 403, however this change in OpenApiSpecStyleValidator should fix the issue:

```
private void validateModelProperties(String modelName, Schema model) {
        if (model.getProperties() != null) {
            model.getProperties().forEach((propertyName, property) -> {
                if (parameters.isValidateModelPropertiesExample() && property.getExample() == null
                        && ((property.getItems() == null && property.getRef() == null)
                            || (property.getItems() != null && property.getItems().getRef() == null))) {
                        errorAggregator.logMissingOrEmptyModelAttribute(modelName, propertyName, "example");
                }
            });
        }
    }

```

**Comment by JFCote** [comment]
Hi @theldberg !

Thank you for reporting this issue. For the problem with the PR, it's probably because you tried to create a PR directly in this project. You need to fork the project and then submit a PR from your fork to this repo. Here is how to do this: https://github.com/openapitools/openapi-generator/wiki/FAQ#how-can-i-submit-a-pr-to-fix-bugs-or-make-enhancements

Let me know if it work. Otherwise, I will try to find some time this week or next week to implement your fix.

Have a good day!

**Comment by theldberg** [comment]
Thanks @JFCote I created the PR from my fork:  https://github.com/OpenAPITools/openapi-style-validator/pull/170

**Comment by JFCote** [comment]
Fixed by #170
